main.rb

```diff
@@ -0,0 +1,28 @@
+require './my_enumerable'
+
+# Create class MyList
+class MyList
+  include MyEnumerable
+  def initialize(*args)
+    @list = args
+  end
+
+  def each(&block)
+    @list.each { |n| block.call(n) }
+  end
+end
+
+# Instance a new list
+list = MyList.new(3, 4, 5, 6)
+
+# Test MyEnumerable methods
+all_numbers = list.all? { |n| n < 6 }
```

Comment:
- [ ] Great job so far @NicolasAndrehh and @sanieni6 However, I noticed that your `#all` method isn't currently returning the right response. From your `list`, the values you provided are all lesser than 6. So your answer for this particular method should return `true` and not `false` :wink: 
- Your solution
![image](https://github.com/sanieni6/Enumerable_exercise/assets/102699958/468fbe7e-21c6-48de-a909-507036e0ceca)

Reply:
Hey @NahnahAJ, thank you so much for the review 😄, but kindly re evaluate it please, because the method is returning the right value because 6 isn't less than 6, that's why the method returns false

Reply:
- [ ] I get you @NicolasAndrehh I guess if that's where you're coming from then that works too. :+1: My apologies then, you can resubmit for approval. I'll leave this here for the next reviewer :100: